swarm_copy/tools/bluenaas_memodel_getone.py

```diff
@@ -24,7 +24,7 @@ class MEModelGetOneMetadata(BaseMetadata):
 class InputMEModelGetOne(BaseModel):
     """Inputs for the BlueNaaS single-neuron simulation."""
 
-    model_id: str = Field(
```

Comment:
Using variables in pydantic classes starting with 'model_' clashes with internal pydantic variables so I had to rename it.